packages/@storefront/core/src/core/defaults.ts

```diff
@@ -91,6 +91,9 @@ const DEFAULTS: Partial<Configuration> = {
   refinements: {
     overrides: {},
   },
+  navigations: {
+    type: {},
```

Comment:
Is the `type` key reserved for 'toggle'/'boolean'-type navigations? If so, is there a more semantic key?

Reply:
type is set up to be future-proofed so we can add additional types of navigations beyond just toggle/boolean-type.